src/staticanalysis/bot/static_analysis_bot/__init__.py

```diff
@@ -165,6 +167,10 @@ def is_third_party(self):
         '''
         Is this issue in a third party path ?
```

Comment:
Maybe we can remove this check and rely on the try tasks to deal with this (they probably already do).

Reply:
Once we are remote only, yes